### DISCUSSION TITLE
Subtitle download

### DISCUSSION THREAD

**Discussion by Name2889** [ErrorFlynn/ytdlp-interface#122] [General]
Hi
Thank you for this GUI. Is it possible to download subtitles with this? It downloads video and audio, but does not see subtitles.
E.g. this URL. Video contains subtitles, and I can download it via ytdlp, but GUI is not displaying it.
https://www.zee5.com/global/tv-shows/details/kumkum-bhagya/0-6-127/khushi-arrives-at-the-malhotra-mansion/0-1-6z5510993
Thank you.

**Comment by ErrorFlynn** [comment]
The GUI doesn't let you select subtitles like it does formats, but you can still use `--sub-langs`, `--sub-format`, and `--write-subs` as custom arguments. I'll see if I can add a subtitle selection window at some point in the future.
